interfaces/cython/cantera/transport.pyx

```diff
@@ -256,6 +272,76 @@ cdef class Transport(_SolutionBase):
         def __get__(self):
             return get_transport_1d(self, tran_getMobilities)
 
+    def get_viscosity_polynomial(self, i):
+        """Get the polynomial fit to the logarithm of temperature for 
+        the viscosity of species i."""
+        n_coeffs = 4 if self.transport.CKMode() else 5
+        return get_transport_polynomial(self, tran_getViscosityPolynomial, i, n_coeffs)
+
+    def get_thermal_conductivity_polynomial(self, i):
+        """Get the polynomial fit to the logarithm of temperature for 
+        the thermal conductivity of species i."""
+        n_coeffs = 4 if self.transport.CKMode() else 5
+        return get_transport_polynomial(self, tran_getConductivityPolynomial, i, n_coeffs)
+
+    def get_binary_diff_coeffs_polynomial(self, i, j):
+        """Get the polynomial fit to the logarithm of temperature for 
+        the binary diffusion coefficient of species i and j."""
+        n_coeffs = 4 if self.transport.CKMode() else 5
+        return get_binary_transport_polynomial(self, tran_getBinDiffusivityPolynomial, i, j, n_coeffs)
+
+    def get_collision_integral_polynomials(self, i, j):
+        """Get the polynomial fit to the logarithm of temperature for 
+        the collision integral of species i and j."""
+        cdef np.ndarray[np.double_t, ndim=1] adata = np.empty(7 if self.transport.CKMode() else 9)
+        cdef np.ndarray[np.double_t, ndim=1] bdata = np.empty(7 if self.transport.CKMode() else 9)
+        cdef np.ndarray[np.double_t, ndim=1] cdata = np.empty(7 if self.transport.CKMode() else 9)
+        self.transport.getCollisionIntegralPolynomial(i, j, &adata[0], &bdata[0], &cdata[0])
+        return adata, bdata, cdata
+
+    def set_viscosity_polynomial(self, i, values):
+        """Set the polynomial fit to the logarithm of temperature for 
+        the viscosity of species i."""
+        if len(values) != (4 if self.transport.CKMode() else 5):
+            msg = "Got {}. Expected {}".format(len(values), (4 if self.transport.CKMode() else 5))
+            raise ValueError('Array has incorrect length. ' + msg + '.')
```

Comment:
it may be easier to read if you assigned the expected length of the list to a new variable, e.g.
```Python
n_values = 4 if self.transport.CKMode() else 5
if len(values) != n_values:
    raise ValueError(
        f"Array has incorrect length: expected {n_values} but received {len(values)}")
```
There are also some additional minor changes above: f-strings make things more readable, and Cantera is now preferring double quotes (`"`) rather than single quotes (`'`) for the definition of strings.

The same comments apply for the remaining functions below.

Reply:
OK